_topic_map.yml

```
@@ -3042,9 +3042,14 @@ Topics:
     # Eventing
   - Name: Creating Knative Eventing components in the Administrator perspective
     File: serverless-cluster-admin-eventing
+#  - Name: Configuring the Knative Eventing custom resource definition
+#    File: knative-eventing-CRD-config
+# Uncomment once we add some configs to this section, adding now for consistency with Serving docs as a placeholder
     # Serving
   - Name: Creating Knative Serving components in the Administrator perspective
     File: serverless-cluster-admin-serving
+  - Name: Configuring the Knative Serving custom resource definition
```

```suggestion
  - Name: Configuring the Knative Serving custom resource
```

The resource definition is the CRD, the blueprint so to say. The CR is the actual object the customer creates.

---

Always confusing these tbh 😅 thanks, I'll correct it